app/javascript/components/modals/contact-us/styles.scss

```diff
@@ -0,0 +1,87 @@
+@import '~styles/settings.scss';
+$mobile-padding: rem(20px);
+$desktop-padding: rem(40px);
+
+.c-form-container {
+  position: relative;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  padding: 3.125rem;
+  padding-top: 2.5rem;
+  border-bottom: 66px solid transparent;
+  outline: none;
+  margin: 0 auto;
+  max-width: 700px;
+  color: #333;
+  border: none;
+  box-shadow: none;
+  background: transparent;
+  border-radius: 0;
+  font-size: 14px;
+  line-height: 1.25;
+
+  .c-button {
+    margin-bottom: 30px;
+  }
+
+  h3 {
+    font-weight: 500;
+    font-size: 1rem;
+    text-transform: uppercase;
+    color: #555;
+    line-height: 1.25;
+  }
+
+  .subtitle {
+    margin-top: 0.3125rem;
+    font-size: 0.8125rem;
+    line-height: 1.2;
+    color: #999;
+    margin-bottom: 1.875rem;
+  }
+}
+
+.feedback-message {
```

Comment:
This isn't scope to the class of the component. If another class is created with the same name it will be polluted.